tests/Core.Tests/WikiStringTests.fs

```diff
@@ -73,6 +73,12 @@ let ``Detects unofficial form - not used``() =
     |> isOfficial
     |> Assert.False
 
+[<Fact>]
+let ``Detects unofficial form - dialect``() = 
+    "znalečná*"
```

Comment:
are you sure it's the right word? test seems identical to the one above

Reply:
Right, copy paste mistake in a sleepy state - corrected now